src/apps/stable/features/playback/utils/mediaSessionSubscriber.ts

```diff
@@ -0,0 +1,178 @@
+import { MediaType } from '@jellyfin/sdk/lib/generated-client/models/media-type';
+
+import { getImageUrl } from 'apps/stable/features/playback/utils/image';
+import { PlaybackSubscriber } from 'apps/stable/features/playback/utils/playbackSubscriber';
+import nowplayinghelper from 'components/playback/nowplayinghelper';
+import type { PlaybackManager } from 'components/playback/playbackmanager';
+import { MILLISECONDS_PER_SECOND, TICKS_PER_MILLISECOND } from 'constants/time';
+import browser from 'scripts/browser';
+import shell from 'scripts/shell';
+import type { ItemDto } from 'types/base/models/item-dto';
+import type { PlayerState } from 'types/playbackStopInfo';
+import type { Event } from 'utils/events';
+
+/** The default image resolutions to provide to the media session */
+const DEFAULT_IMAGE_SIZES = [96, 128, 192, 256, 384, 512];
+
+const hasNavigatorSession = 'mediaSession' in navigator;
+const hasNativeShell = !!window.NativeShell;
+
+const getArtwork = (item: ItemDto): MediaImage[] => {
+    const artwork: MediaImage[] = [];
+
+    DEFAULT_IMAGE_SIZES.forEach(height => {
```

Comment:
This can be a map

Reply:
I didn't use 'map' since it's not 1:1 (`getImageUrl` can return `null` in which case we skip that entry)